build_tools/github_action/configure_ci.py

```diff
@@ -127,12 +127,12 @@ def check_for_non_skippable_path(paths: Optional[Iterable[str]]) -> bool:
     return any(not is_path_skippable(p) for p in paths)
 
 
-# TODO(#199): rename all of these to `ci_*.yml` so this is easier to understand?
 GITHUB_WORKFLOWS_CI_PATTERNS = [
-    "ci.yml",
     "setup.yml",
-    "build_*_packages.yml",
-    "test_*_packages.yml",
+    "ci*.yml",
+    "build*package*.yml",
+    "test*packages.yml",
+    "test*.yml",  # This may be too broad, but there are many test workflows.
```

Comment:
Notably this will cover `test_pytorch_docker.yml` if https://github.com/ROCm/TheRock/pull/578 goes through without a different name. We could add an exclude pattern on top of this include pattern if we care enough.

We could list all the workflows explicitly, or name the "ci" workflows `test_ci_*.yml`.